ara/plugins/callback/ara_default.py

```diff
@@ -838,10 +838,17 @@ def _load_result(self, result, status, **kwargs):
 
         # Sanitize facts
         if "ansible_facts" in results:
-            for fact in self.ignored_facts:
-                if fact in results["ansible_facts"]:
-                    self.log.debug("Ignoring fact: %s" % fact)
-                    results["ansible_facts"][fact] = "Not saved by ARA as configured by 'ignored_facts'"
+            ignored_facts_hint = "Not saved by ARA as configured by 'ignored_facts'"
+
+            if "all" in self.ignored_facts:
+                self.log.debug("Ignoring all facts")
+                results["ansible_facts"] = {"all": ignored_facts_hint}
```

Comment:
I kind of like that we don't iterate across every fact to set the values individually. Good thinking.